pkg/pluginmanager/manager.go

```diff
@@ -733,7 +737,23 @@ func installOrUpgradePlugin(p *discovery.Discovered, version string, installTest
 	}
 
 	// Log message based on different installation conditions
-	logPluginInstallationMessage(p, version, plugin != nil, isPluginAlreadyInstalled)
+	installingMsg, installedMsg := getPluginInstallationMessage(p, version, plugin != nil, isPluginAlreadyInstalled)
+
+	var spinnerErr error
+	var spinner component.OutputWriterSpinner
+	if enableSpinner {
+		// Set spinner options
+		spinnerOptions := component.OutputWriterSpinnerOptions{
+			OutputWriterSpinnerOptions: []component.OutputWriterSpinnerOption{
+				component.NoEmptylineAfterSpinner(),
+			},
+		}
+		// Initialize the spinner
+		spinner, spinnerErr = component.NewOutputWriterSpinnerWithSpinnerOptions(os.Stdout, component.TableOutputType, installingMsg, true, spinnerOptions)
```

Comment:
The other printouts during installation are on `os.Stderr`, so, although the spinner disappears, I think it makes more sense to print the spinner on `os.Stderr` also